Resources/doc/reference/installation.rst

```diff
@@ -0,0 +1,94 @@
+Installation
+============
+
+SonataArticleBundle can be installed at any moment during a project's lifecycle,
+whether it's a clean Symfony installation or an existing project.
+
+Downloading the code
+--------------------
+
+Use composer to manage your dependencies and download SonataArticleBundle:
+
+.. code-block:: bash
+
+    $ php composer.phar require sonata-project/article-bundle
+
+You'll be asked to type in a version constraint. 'dev-master' will get you the latest
+version, compatible with the latest Symfony version. Check `packagist <https://packagist.org/packages/sonata-project/article-bundle>`_
```

Comment:
I don't think you will be asked anything, and if you are, let's not recommend `dev-master`, ok?